compat_policy.go

```diff
@@ -0,0 +1,89 @@
+package nftables
+
+import (
+	"fmt"
+
+	"github.com/google/nftables/expr"
+	"golang.org/x/sys/unix"
+)
+
+const NFT_RULE_COMPAT_F_INV uint32 = (1 << 1)
+const NFT_RULE_COMPAT_F_MASK uint32 = NFT_RULE_COMPAT_F_INV
+
+// Used by xt match or target like xt_tcpudp to set compat policy between xtables and nftables
+// https://elixir.bootlin.com/linux/v3.13/source/net/netfilter/nft_compat.c#L126
```

Comment:
Is there any reason you’re linking to v3.13 specifically? That’s quite an old Linux version by now. Could you link to the latest stable one instead?

Reply:
because 3.13 is the first version that kernel bring nftables into it. so think about compat, i usually use this version, but change kernel version is ok, because it actually don't change